manual/manual.tex

```diff
@@ -103,7 +103,8 @@
 
   \vfill
 
-  \copyright~{\commityear} General Dynamics C4 Systems.\\
+  \copyright~{2014} General Dynamics C4 Systems.\\
+  \copyright~{\commityear} seL4 Foundation.\\
```

Comment:
This is not quite right. The copyright year does not novate automatically with the last commit (must be substantial changes), and the copyright of the manual is not with the foundation.

There have been many substantial non-GD contributions in the past, so we could say something more generic like
```suggestion
  \copyright\ the seL4 authors and contributors.\\
```

Reply:
* will add your suggestion.
* Can you confirm 2014 for GD is ok. Should this list a starting year also? Should we add 2020 as starting year for the seL4 Foundation?
* Should we add a license note, can be takle "Distributed under the Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0) License." that is used in the seL4 whitepaper. 
* Should we add the statement "seL4 is a trademark of LF Projects, LLC.", as the whitepaper says?

Reply:
- 2014 is correct for GD, but I'd leave out both GD and the Foundation, actually, and only do the generic part, since they would both be authors/contributors
- unfortunately can't do CC-BY-SA-4.0 (see other comment below)
- yes, the trademark sentence is definitely a good idea